### PR TITLE
Show standards on lesson edit page

### PR DESCRIPTION
**Pull request by davidsbailey**
Starts [PLAT-328]. This is how it looks:

Screen Shot 2021-03-24 at 11 21 03 PM

The delete button is not yet hooked up to anything. That will come in https://github.com/code-dot-org/code-dot-org/pull/39706.

## Testing story

Minor updates to existing tests. More tests to be added once we start adding/deleting standards.

## Future work

This doesn't have all the columns shown in the spec:
Screen Shot 2021-03-25 at 8 31 53 AM

When I tried this, it seemed off because some of the category descriptions are quite long. One idea instead is to show a live preview next to it, like we do for activity sections:
Screen Shot 2021-03-25 at 8 37 19 AM

This is left as future work: https://codedotorg.atlassian.net/browse/PLAT-882

[PLAT-328]: https://codedotorg.atlassian.net/browse/PLAT-328

### CODE REVIEW

File: apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    justifyContent: 'space-evenly',
+    backgroundColor: 'white'
+  },
+  remove: {
+    fontSize: 14,
+    color: 'white',
+    background: color.dark_red,
+    cursor: 'pointer',
+    textAlign: 'center',
+    width: '50%',
```
Comment (dmcavoy) [inline apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx]: Are you leaving this at 50% because you will be adding the add button in the next PR?

**Review by dmcavoy — APPROVED** [review]
Looks good to me. I like this as an initial stopping point. I might try this with the blocks PR I have open

File: apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx]: oops - copy paste error. there were two buttons in the place I copied this from. I will fix. no current plans to add an "add" button soon. we'll use the dropdowns for that.

Comment (davidsbailey) [reply apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx]: I changed the styles to get rid of some of the whitespace:
Screen Shot 2021-03-25 at 11 02 55 AM

**Pull request merged**